docs/user-guide/configuration/stages.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: main
+title: Stages
+category: User Guide
+menu: menu
+toc:
+  - title: Example
+    url: "#example"
+  - title: Setup and Teardown
+    url: "#setup-and-teardown"
+  - title: Limitations
+    url: "#limitations"
+---
+
+# Pipeline Stages
+
+The configuration of stages is employed to categorize one or more jobs that execute steps to achieve a similar objective. For instance, jobs such as `ci-deploy`, `ci-test`, and `ci-certify` can be grouped together under a stage named integration.
```

Comment:
Just a suggestion. Nothing really wrong with what you're saying, but I think a simpler use of language could be easier to read for users (and a little more consistent with what we already have). 

```suggestion
Stages can be used to categorize one or more jobs that have steps with similar objectives. For instance, jobs `ci-deploy`, `ci-test`, and `ci-certify` can be grouped together under a stage named `integration`.
```